packages/opencode/README.md

```
@@ -165,6 +263,220 @@ bun run build
+| Tool                | Description                                 |
+| ------------------- | ------------------------------------------- |
+| `background_task`   | Launch an agent task in the background      |
+| `background_output` | Retrieve the result of a completed task     |
+| `background_cancel` | Cancel a running or pending background task |
+
+### Usage
+
+```typescript
+// Launch a background task
+background_task({
+	agent: 'scout',
```

markdownlint-cli2 (0.20.0)

[warning] Hard tabs
Column: 1

(MD010, no-hard-tabs)